star-navigation/src/components/right-container/right-container.tsx

```diff
@@ -30,6 +35,20 @@ export const RightContainer: React.FC<IProps> = ({ inputState, disableInputs, se
 
   const handleJourneyChangeBtoC = () => setInputState({ journeyLeg: "BtoC" });
 
+  const handleTakeYourRouteClick = () => setInputState({ showUserTrip: true });
+
+  const handleResetRouteClick = () => setInputState({
+    pointADepartureSnapshot: null,
+    pointBArrivalSnapshot: null,
+    pointBDepartureSnapshot: null,
+    pointCArrivalSnapshot: null,
+    showUserTrip: false,
+    journeyLeg: "AtoB",
+    timeOfDay: 0
+  });
+
+  const handlePointBChange = (pointB: { x: number; y: number }) => setInputState({ pointB });
```

Comment:
Should use `useCallback()` here since this function is a `useEffect` dependency in the `RouteMap` component below.